### DISCUSSION TITLE
Play LottieAnimation manually according value

### DISCUSSION THREAD

**Discussion by arthurBusarov** [airbnb/lottie-ios#2217] [General]
Hello there.
I have nex scenario - I have a scroll view with lottie animation on top. When user scrolls the scrollView down (the scroll offset increases) the animation should be played manually related on the offset value.
In other words, I have a spinner animation, for pull to refresh, when user scrolls (drags) scroll view the animation have to make a full round related on offset value and if it happens the refresh could start and animation have to play in normal mode
`.playing(loopMode: .loop)` 
But how could I do this manual scroll playing in swiftUI?
I found [this](https://lottiefiles.com/interactivity) and it could be kinda _Sync Lottie with scroll_ but I cannot find the same for the swiftUI.

**Comment by nikolaydubina** [comment]
hey, did you find the answer?

**Reply by arthurBusarov** [reply]
No, unfortunatelly.
